evalai/utils/challenges.py

```diff
@@ -174,3 +174,52 @@ def display_participated_or_hosted_challenges(is_host=False, is_participant=Fals
                 pretty_print_challenge_data(challenge)
         else:
             echo("Sorry, no challenges found!")
+
+
+def pretty_print_challenge_phase_split_data(splits):
+    """
+    Function to print the details of a Challenge Phase Splits.
+    """
+    for split in splits:
+        br = style("----------------------------------------"
+                   "--------------------------", bold=True)
+
+        dataset_split_name = "\n{}".format(style(split["dataset_split_name"],
+                                           bold=True, fg="green"))
```

Comment:
The 2nd line should line up with the arguments to `style()`. This formatting makes it seem like it is an arg for `format()`.